packages/storybook8/stories/Composites/TypeScriptLoading/Docs.mdx

```diff
@@ -0,0 +1,145 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Composites/JavaScript Loaders" />
+
+# Using the UI Composites in other Frameworks
+
+The Azure Communication Services UI library is written in [React](https://react.dev/). If you are using a different framework,
+you can still use the UI Composites by using our Loader functions for each of the composites.
+
+These functions mount each of our React Composites to your applications DOM. The Functions that are available are:
+
+- `loadChatComposite`
+- `loadCallComposite`
+- `loadCallWithChatComposite`
+- `loadOutboundCallComposite`
+
+## How they work
+
+Since all different [Node](https://nodejs.org/en) frameworks are just JavaScript, CSS, and HTML you can use each of the
```

Comment:
React isn't really a `NodeJs framework`, it's a front-end library. When you bundle tools like redux it start to act like a framework... but even then people argue over the term.
Perhaps we could say something more like
```
Our JavaScript loaders use [react]() under the hood. They handle all of the react complexity and allow the composites to be placed in any application, including applications written with JQuery, Angular, Blazor etc.

They work by taking in a DOM element. They render and attach the React Composite to that element. This means
that you can use the UI Composites in any framework that can render HTML elements to the DOM.

**Note:** Something to keep in mind when using these loader functions .....
```